docs/reference/commandline/stack_config.md

```diff
@@ -26,7 +26,7 @@ Options:
       --help            Print usage
 ```
 
-Displays the configuration of a stack.
+Display the configuration of a stack.
```

Comment:
It looks as if it is in the imperative form.  So IMO you don't need to change this.